### ISSUE TITLE
The pageLoadTimeout option doesn't work

### ISSUE THREAD

**Issue by felis2803** [DevExpress/testcafe#7437]
### What is your Scenario?

I'm running tests on a page that takes a long time to load. I use the `pageLoadTimeout` option to delay running tests until the page is loaded.

### What is the Current behavior?

Tests are run without waiting for either the timeout specified by the pageLoadTimeout option or the `window.onload` event.

### What is the Expected behavior?

Tests should be run within the specified timeout, according to the [documentation](https://testcafe.io/documentation/402655/reference/testcafe-api/runner/run#pageloadtimeout).

### What is your TestCafe test code?

```js
import { ClientFunction } from 'testcafe';

fixture `New Fixture`
    .page `localhost:3000`;

test(`New Test`, async t => {
    await ClientFunction(() => console.log('TestCafe test started:', Date.now()))();

    await t.debug();
});
```

### Your complete test report
![изображение](https://user-images.githubusercontent.com/34184692/209139887-fd05a0e7-eb04-44e1-8f83-b17d17d14e1e.png)

### Steps to Reproduce

`git clone https://github.com/felis2803/testcafe-temp-9`
`cd testcafe-temp-9`
`npm i`
`npm test`

Once the browser is launched, open the DevTools console.
Watch for messages in the console. There should be 4 of them.
If pageLoadTimeout is working properly, the test launch message should be the last one.
However, you may notice that it comes second.

### TestCafe version

2.1.0

### Node.js version

16.18.0

**Comment by miherlosev** [comment]
The `pageLoadTimeout`option will be applied only if there are user-defined handlers for the `window.load` event.
See tests [here](https://github.com/DevExpress/testcafe/tree/master/test/functional/fixtures/api/es-next/page-load-timeout).

Need to fix the option description in the documentation.

**Comment by titerman** [comment]
Fixed with the release of TestCafe v2.6

> TestCafe applies the timeout when the user delays test execution until the window.loadEventRaised event.

https://testcafe.io/documentation/402835/guides/basic-guides/debug-tests#page-load-timeout
